database/seeds/ProductTypeSeeder.php

```diff
@@ -0,0 +1,141 @@
+<?php
+
+use Illuminate\Database\Seeder;
+use Illuminate\Support\Facades\DB;
+
+class ProductTypeSeeder extends Seeder
+{
+    /**
+     * Run the database seeds.
+     *
+     * @return void
+     */
+    public function run()
+    {
+        DB::table('product_types')->insert([
+            'type_nl' => 'Soep',
+            'type_en' => 'Soup',
+            'hasRice' => 0,
```

Comment:
```suggestion
            'has_rice' => 0,
```

Maar dat geldt dan ook voor alle andere